src/main/java/controllers/TableViewController.java

```diff
@@ -98,12 +95,25 @@ private void fillTable(DataTable table) {
 	 * @param columns A List containing the DataColumns
 	 */
 	private void fillTableHeaders(List<DataColumn> columns) {
-		for (int i = 0; i < columns.size(); i++) {
+		int i = 0;
+		while (i < columns.size()) {
```

Comment:
In line 106 heb ik de i weer nodig om een extra colom op de index erna toe te voegen